config/routes.rb

```diff
@@ -2,6 +2,7 @@
   root                  to: "pages#home"
   get  "/about",        to: "pages#about"
   get  "/testimonials", to: "pages#testimonials"
+  get "/guest/sessions", to: "pages#home"
```

Comment:
I'd keep this in thee guest namespace to keep things neater.

```ruby
namespace :guest do
  # ...
  get "/sessions", to: "pages#home"
end
```
this should work there.